### ISSUE TITLE
About "Ignore list"

### ISSUE THREAD

**Issue by chtiland** [samwilson/dokuwiki-plugin-log404#5]
Hi !
- When ignoring error is still displayed
- Is there a way to managed ignored errors ?

Thanks

**Comment by mekineer** [comment]
Yes, I'm not sure what "add to ignore list" does.  I can't find a file that contains the ignored elements.  I think Log404.csv just contains the current viewed list?
